src/content/docs/nrql/using-nrql/funnels-evaluate-data-series-related-events.mdx

```diff
@@ -40,23 +38,21 @@ FROM DATA_TYPE
   SINCE TIMEFRAME
 ```
 
-See also a more complex [funnel query example](#example).
-
 Funnel queries require the [`funnel` function](/docs/query-data/nrql-new-relic-query-language/getting-started/nrql-syntax-components-functions#func-funnel), an [attribute](/docs/using-new-relic/welcome-new-relic/get-started/glossary#attribute) to funnel, and at least two steps:
 
-1. The first step is the anchor step, which will always represent 100% of the results.
-2. The second and later steps describe the number of users who have also completed additional actions. This number typically will be less than 100%. However, it could be 100% if every user who completes action (A) also completes the additional actions being queried.
+1. The anchor step, which always represents 100% of the results.
+2. One or more steps that describe the number of users who have also completed additional actions, typically than the 100% from the anchor step. However, it could be 100% if every user who completes action (A) also completes the additional actions you're querying.
 
 ## Funnel query technical details
 
-One way to use funnel queries is to calculate the rate at which multi-step, ordered sequences were completed over a given timeframe, like in the New Relic signup [example](#why-matters). But you can also use funnels to calculate overlap between several distinct actions that don't have a sequential relationship to one another. In other words, the order of steps doesn't impact the calculations performed.
+You can use funnel queries to calculate the completion rate for which multi-step, ordered sequences over a given timeframe, like in the New Relic signup [example](#why-matters). You can also use funnels to calculate overlap between several distinct actions that don't have a sequential relationship to one another. In other words, the order of steps won't impact the calculations performed.
```

Comment:
I can't quite make sense of that first sentence there.

Reply:
also, the end of the paragraph is too passive. How about simply: "the order of steps won't impact the calculations." (the performed being implied)